src/bindgen/ir/function.rs

```diff
@@ -246,6 +258,10 @@ impl Source for Function {
                 write!(out, " {}({})", swift_name_macro, func.swift_name());
             }
 
+            if func.never_return {
+                out.write(" CBINDGEN_NO_RETURN_ATTR ");
```

Comment:
here instead of hardcoding this we'd do something like:

```rust
if func.never_return {
    if let Some(ref no_return_attr) = config.function.no_return {
        out.write(" ");
        out.write(no_return_attr);
        out.write(" ");
    }
}
```